### PR TITLE
Ycash support

### PR DESCRIPTION
**Pull request by zzzpotato**
Mainnet is not active yet

### CODE REVIEW

**Comment by egyptianbman** [comment]
@zzzpotato Can you go ahead and add the config file for mainnet?

**Comment by egyptianbman** [comment]
Alternatively, we can merge this and you can create a new branch for the mainnet config.

**Review by egyptianbman — DISMISSED** [review]
Thanks!

**Comment by zzzpotato** [comment]
I added the mainnet configs and renamed the PR

File: coins/yec.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        "s1Y63AsWsJTk5t5nSZfaFcFWmtfnFUUAu2V",
+        "s1Y2U4GsfZdP9LAbC97GAmSdihBX5FU9gQn",
+        "s1bPYWZMXzyN2ML2vswDiCckmas775QFs2Q",
+        "s1erG25RcWYCiBPbT7khTU4ULhzm8jJZ7pv",
+        "s1kYEiPdFZ3oV389q2MmSYY932qPF1ygVtx"
+    ],
+
+    "txfee": 0.0001,
+    "sapling": true,
+    "peerMagic": "aae83f5f",
```
Comment (egyptianbman) [inline coins/yec.json]:
```suggestion
    "peerMagic": "24e92764",
```

File: coins/testnet/yec.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        "smZyciv3CGZLAFU8d2yKff33FU8f2nek5yx",
+        "smZzRpCLENnxN5JgMHaKtMruTCi7jsa7Tak",
+        "smaWKSqvUJvajRquGbaHBdNbL78fDf6hdwE",
+        "smazMQ9G7NLJXAzX4ZMKc8x6DigyeoEucgk",
+        "smbTaZmsKNEVstoQVyZcJAMJExiytfnwaMU"
+    ],
+
+    "txfee": 0.0001,
+    "sapling": true,
+    "peerMagic": "aae83f5f",
```
Comment (egyptianbman) [inline coins/testnet/yec.json]:
```suggestion
    "peerMagic": "fa1af9bf",
```

**Review by egyptianbman — CHANGES_REQUESTED** [review]

**Review by egyptianbman — APPROVED** [review]
Thanks!

**Pull request merged**